bitnami/grafana-mimir/values.yaml

```diff
@@ -0,0 +1,4185 @@
+  ## BlockStorage config for mimir pods
+  ## ref: https://grafana.com/docs/mimir/latest/operators-guide/configure/reference-configuration-parameters/#alertmanager_storage
+  ## @param mimir.blockStorage.backend Backend storage to use
+  ## @param mimir.blockStorage.config Configures connection to the backend store
+  ## e.g.:
+  ## config:
+  ##   access_key_id: AKIAMYACCESSKEY123456789
+  ##   secret_access_key: MYSECRETACCESSKEY
+  ##   region: us-east-1
+  ##   bucket_name: mimir
+  ##   endpoint: s3.us-east-1.amazonaws.com
+  ##   insecure: false
+  ##
+  blockStorage:
+    backend: s3
+    config: {}
+
+
+## @section Alertmanager Deployment Parameters
+##
+alertmanager:
+  ## @param alertmanager.enabled Enable alertmanager deployment
+  ##
+  enabled: false
+  ## @param alertmanager.extraEnvVars Array with extra environment variables to add to alertmanager nodes
+  ## e.g:
+  ## extraEnvVars:
+  ##   - name: FOO
+  ##     value: "bar"
+  ##
+  extraEnvVars: []
+  ## @param alertmanager.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for alertmanager nodes
+  ##
+  extraEnvVarsCM: ""
+  ## @param alertmanager.extraEnvVarsSecret Name of existing Secret containing extra env vars for alertmanager nodes
+  ##
+  extraEnvVarsSecret: ""
+  ## @param alertmanager.command Override default container command (useful when using custom images)
+  ##
+  command: []
+  ## @param alertmanager.args Override default container args (useful when using custom images)
+  ##
+  args: []
+  ## @param alertmanager.extraArgs Add additional args to the default container args (useful to override configuration)
+  ## e.g:
+  ## extraArgs:
+  ##   - "-log.level=debug"
+  extraArgs: []
+  ## @param alertmanager.replicaCount Number of Alertmanager replicas to deploy
+  ##
+  replicaCount: 1
+  ## @param alertmanager.podManagementPolicy Statefulset Pod management policy, it needs to be Parallel to be able to complete the cluster join
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
+  ##
+  podManagementPolicy: OrderedReady
+  ## Configure extra options for Alertmanager containers' liveness, readiness and startup probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes
+  ## @param alertmanager.livenessProbe.enabled Enable livenessProbe on Alertmanager nodes
+  ## @param alertmanager.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param alertmanager.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param alertmanager.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param alertmanager.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param alertmanager.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    failureThreshold: 3
+    initialDelaySeconds: 60
```

Comment:
There is no container logic, does it really need such a huge `initialDelaySeconds`?

Reply:
I've just reduced to 20